<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import data</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-open"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="pathEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Select path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="openButton">
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset theme="document-open">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="jobCombo"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="docCombo"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>openClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>pathChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openClicked()</slot>
  <slot>pathChanged(QString)</slot>
 </slots>
</ui>
